### ISSUE TITLE
Template Chooser: Search doesn't find existing site to copy as template

### ISSUE THREAD

**Issue by bree-z** [cuny-academic-commons/commons-in-a-box#422]
Hi Boone, 

When I try to create a new site template based on an existing site, I can't find the existing site with the search form. 

On https://openlabdev.commonsinabox.org/wp-admin/post-new.php?post_type=cboxol_site_template I tried to create a template based on this site: https://openlabdev.commonsinabox.org/mat-1575-openlab-model-course/

I first tried searching with 'mat-1575-openlab-model-course' and then tried a few other things like 'MAT 1575', 'MAT', and then with the site ID 588. I tried some other site URLs and saw the same thing -- the text 'Searching...' appears, but it doesn't find anything.

Thanks!

<img width="567" alt="Screen Shot 2022-12-27 at 5 26 37 AM" src="https://user-images.githubusercontent.com/7189824/209669302-30dadf6b-7aef-4def-b68e-4733b289b902.png">

**Comment by boonebgorges** [comment]
The issue is that we are using WP's internal `get_sites()`. Its `search` parameter matches only the `domain` and `path` columns. This is a limitation of WP's tooling that is not likely to be resolved soon.

I have made modifications such that the 'blogname' value is mirrored to the global blogmeta table, and there's a filter on the core query so that the 'blogname' meta value is matched. This change also requires an Upgrade routine to add 'blogname' to blogmeta for existing sites. I've deployed to openlabdev.commonsinabox.org and run the upgrade routine and search should now be working as expected.
